arangosh/Import/ImportFeature.cpp

```diff
@@ -269,73 +275,109 @@ void ImportFeature::start() {
     }
   }
 
-  std::unique_ptr<SimpleHttpClient> httpClient;
-
   try {
-    httpClient = client->createHttpClient();
+    _httpClient = client->createHttpClient();
   } catch (...) {
     LOG_TOPIC(FATAL, arangodb::Logger::FIXME)
         << "cannot create server connection, giving up!";
     FATAL_ERROR_EXIT();
   }
 
-  httpClient->params().setLocationRewriter(static_cast<void*>(client),
+  _httpClient->params().setLocationRewriter(static_cast<void*>(client),
+
                                            &rewriteLocation);
-  httpClient->params().setUserNamePassword("/", client->username(),
+  _httpClient->params().setUserNamePassword("/", client->username(),
                                            client->password());
 
   // must stay here in order to establish the connection
-  httpClient->getServerVersion();
 
-  if (!httpClient->isConnected()) {
+  int err = TRI_ERROR_NO_ERROR;
+  auto versionString = _httpClient->getServerVersion(&err);
+  auto dbName = client->databaseName();
+  bool createdDatabase = false;
+  
+  auto successfulConnection = [&](){
+    std::cout << "Connected to ArangoDB '"
+              << _httpClient->getEndpointSpecification() << "', version "
+              << versionString << ", database: '"
+              << client->databaseName() << "', username: '" << client->username()
+              << "'" << std::endl;
+
+    std::cout << "----------------------------------------" << std::endl;
+    std::cout << "database:               " << client->databaseName()
+              << std::endl;
+    std::cout << "collection:             " << _collectionName << std::endl;
+    if (!_fromCollectionPrefix.empty()) {
+      std::cout << "from collection prefix: " << _fromCollectionPrefix
+                << std::endl;
+    }
+    if (!_toCollectionPrefix.empty()) {
+      std::cout << "to collection prefix:   " << _toCollectionPrefix << std::endl;
+    }
+    std::cout << "create:                 " << (_createCollection ? "yes" : "no")
+              << std::endl;
+    std::cout << "created database:       " << (_createDatabase ? "yes" : "no")
```

Comment:
should be "create database" (not "created") because this is printed before the database will be created

Reply:
successfulConnection() is called after tryCreateDatabase()

Reply:
true. my fault.